data/block/proto/metaBlock.proto

```diff
@@ -70,9 +70,11 @@ message Economics {
 	bytes  TotalToDistribute      = 2 [(gogoproto.casttypewith) = "math/big.Int;github.com/ElrondNetwork/elrond-go/data.BigIntCaster"];
 	bytes  TotalNewlyMinted       = 3 [(gogoproto.casttypewith) = "math/big.Int;github.com/ElrondNetwork/elrond-go/data.BigIntCaster"];
 	bytes  RewardsPerBlockPerNode = 4 [(gogoproto.casttypewith) = "math/big.Int;github.com/ElrondNetwork/elrond-go/data.BigIntCaster"];
-	bytes  NodePrice              = 5 [(gogoproto.casttypewith) = "math/big.Int;github.com/ElrondNetwork/elrond-go/data.BigIntCaster"];
-	uint64 PrevEpochStartRound    = 6;
-	bytes  PrevEpochStartHash     = 7;
+	bytes  RewardsForCommunity    = 5 [(gogoproto.casttypewith) = "math/big.Int;github.com/ElrondNetwork/elrond-go/data.BigIntCaster"];
+	bytes  CommunityAddress       = 6;
```

Comment:
There is no need for this to appear in the metablock. It is enough to appear in the generated reward transactions

Reply:
the bigInt of rewards for community can remain as it is super visible, but the address can be read from the config.

Reply:
done